### DISCUSSION TITLE
Metadata translations issue

### DISCUSSION THREAD

**Discussion by nikolabul** [open-sdg/open-sdg#1763] [Q&A] [answered]
Hi @brockfanning. I started importing data for indicators and it worked fine for a while but now I encountered a problem with metadata translations. I have some custom metadata fields which are not translated and instead the corresponding translation keys are displayed. I am sure it worked fine a few days ago.

You can see the problem in the screenshot:

![image](https://user-images.githubusercontent.com/84736274/182680576-7e7fe5cf-ecc6-47c9-926a-b38f21a2bd5e.png)

All these translation keys are present in my translations folder and in metada_schema.yml.

**Comment by brockfanning** [comment, marked as answer]
@nikolabul I think that means that the system treats `''` as if it didn't exist, and so it just prints the translation key. In other words, for example, as soon as you add content in for [1-1-1-national-methodology here](https://github.com/montestat/data/blob/develop/translations/en/metadata_texts.yml#L2) that content will replace the translation key on the indicator page.

**Reply by nikolabul** [reply]
@brockfanning thanks for suggestion. I tried what you mentioned but nothing changed.

`1-1-1-national-methodology: 'test test test'`

All the fields from metadata_texts.yml populated with content appear only as translation keys.

**Reply by brockfanning** [reply]
@nikolabul Ah sorry, I was wrong about that. It looks like there is a YAML syntax issue. There are some warnings about it here: https://github.com/montestat/data/runs/7672044933?check_suite_focus=true#step:5:14

One example is [this line](https://github.com/montestat/data/blob/develop/translations/en/metadata_texts.yml#L57) that I think might be caused by the apostrophe in `country's`. You could try wrapping that metadata in `"` instead of `'`.

It's unfortunate that the build succeeds even when there is a problem like that - it would have been better if the whole build would have failed, so it is not so mysterious. If you are able it would be helpful to report that in our Issues section, that invalid YAML syntax in translations does not cause a build failure.

**Reply by nikolabul** [reply]
Yeah, you are right @brockfanning. If I mix ' and " in the same metadata field, then the string breaks. It's a common mistake and I didn't know that this syntax is valid for YAML as well. It can only be seen in the warnings that you mentioned. I fixed all of them and will pay more attention in future. I will report this in Issues section.
